components/collection/activity/Activity.vue

```diff
@@ -4,7 +4,7 @@
     <div ref="wrapper" class="w-full mt-5">
       <div v-if="tablet">
         <div class="columns">
-          <div class="column is-two-thirds">
+          <div class="column is-two-thirds mt-2">
```

Comment:
😕 margin on a column is a bad idea

Reply:
1. why?
2. 
>Better solution would be to change aboslute position on dropdown 
yes, did that https://github.com/kodadot/nft-gallery/pull/7859/commits/1e2694d1f495bcb20e681b316bac862131f88360
and...

https://github.com/kodadot/nft-gallery/pull/7859#issuecomment-1783738246

Reply:
please could you move `mt-2` to the content (`ActivityChart` component) so we wouldn't have futur side effects

Reply:
done